modules/virt-creating-configmap.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-creating-configmap_{context}']
+= Creating a custom operating system ConfigMap
+
+If the operating system of the Red Hat Virtualization virtual machine does not exist in the default ConfigMap of the `vm-import-controller` Custom Resource (CR), you can create a custom ConfigMap to map the operating system to a common template. The template is used to create the new virtual machine in {VirtProductName}.
+
+.Procedure
+
+. Create a file named `configmap-file.txt` that maps the RHV virtual machine's operating system to a common template, as in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default <1>
+data:
+  guestos2common: | <2>
+    "Red Hat Enterprise Linux Server": "rhel"
+    "CentOS Linux": "centos"
+    "Fedora": "fedora"
+    "Ubuntu": "ubuntu"
+    "openSUSE": "opensuse"
+  osinfo2common: | <3>
+    "rhel_6_9_plus_ppc64": "rhel6.9"
+    "rhel_6_ppc64": "rhel6.9"
+    "rhel_6": "rhel6.9"
+    "rhel_6x64": "rhel6.9"
+    "rhel_7_ppc64": "rhel7.7"
+    "rhel_7_s390x": "rhel7.7"
+    "rhel_7x64": "rhel7.7"
+    "rhel_8x64": "rhel8.1"
+    "sles_11_ppc64": "opensuse15.0"
+    "sles_11": "opensuse15.0"
+    "sles_12_s390x": "opensuse15.0"
+    "ubuntu_12_04": "ubuntu18.04"
+    "ubuntu_12_10": "ubuntu18.04"
+    "ubuntu_13_04": "ubuntu18.04"
+    "ubuntu_13_10": "ubuntu18.04"
+    "ubuntu_14_04_ppc64": "ubuntu18.04"
+    "ubuntu_14_04": "ubuntu18.04"
+    "ubuntu_16_04_s390x": "ubuntu18.04"
+    "windows_10": "win10"
+    "windows_10x64": "win10"
+    "windows_2003": "win10"
+    "windows_2003x64": "win10"
+    "windows_2008R2x64": "win2k8"
+    "windows_2008": "win2k8"
+    "windows_2008x64": "win2k8"
+    "windows_2012R2x64": "win2k12r2"
+    "windows_2012x64": "win2k12r2"
+    "windows_2016x64": "win2k16"
+    "windows_2019x64": "win2k19"
+    "windows_7": "win10"
+    "windows_7x64": "win10"
+    "windows_8": "win10"
+    "windows_8x64": "win10"
+    "windows_xp": "win10"
+----
+<1> Optional: Update the namespace.
+<2> Specify the operating system. `guestos2common` maps the operating system to a common template.
+<3> Specify the operating system version. `osinfo2common` maps the operating system resource of the source provider to a common template.
+
+. Create a custom ConfigMap:
++
+----
+$ oc create configmap os-configmap --from-file=./configmap-file.txt
```

Comment:
This command should be replaced with : 
`oc apply -f configmap-file.txt` since the file contains the entire config map definition.
I'd also suggest to rename to  `configmap-file.yaml` to reflect its actual content.
or change it to:
```
cat <<EOF | oc create -f -
----
apiVersion: v1
kind: ConfigMap
metadata:
  name: os-configmap
  namespace: default <1>
data:
  guestos2common: | <2>
    "Red Hat Enterprise Linux Server": "rhel"
    "CentOS Linux": "centos"
    "Fedora": "fedora"
    "Ubuntu": "ubuntu"
    "openSUSE": "opensuse"
  osinfo2common: | <3>
    "rhel_6_9_plus_ppc64": "rhel6.9"
    "rhel_6_ppc64": "rhel6.9"
    "rhel_6": "rhel6.9"
    "rhel_6x64": "rhel6.9"
    "rhel_7_ppc64": "rhel7.7"
    "rhel_7_s390x": "rhel7.7"
    "rhel_7x64": "rhel7.7"
    "rhel_8x64": "rhel8.1"
    "sles_11_ppc64": "opensuse15.0"
    "sles_11": "opensuse15.0"
    "sles_12_s390x": "opensuse15.0"
    "ubuntu_12_04": "ubuntu18.04"
    "ubuntu_12_10": "ubuntu18.04"
    "ubuntu_13_04": "ubuntu18.04"
    "ubuntu_13_10": "ubuntu18.04"
    "ubuntu_14_04_ppc64": "ubuntu18.04"
    "ubuntu_14_04": "ubuntu18.04"
    "ubuntu_16_04_s390x": "ubuntu18.04"
    "windows_10": "win10"
    "windows_10x64": "win10"
    "windows_2003": "win10"
    "windows_2003x64": "win10"
    "windows_2008R2x64": "win2k8"
    "windows_2008": "win2k8"
    "windows_2008x64": "win2k8"
    "windows_2012R2x64": "win2k12r2"
    "windows_2012x64": "win2k12r2"
    "windows_2016x64": "win2k16"
    "windows_2019x64": "win2k19"
    "windows_7": "win10"
    "windows_7x64": "win10"
    "windows_8": "win10"
    "windows_8x64": "win10"
    "windows_xp": "win10"
EOF
```